result/lldpneighbors.yaml

```diff
@@ -0,0 +1,230 @@
+            This field identifies the format and source of the chassis identifier string. 
+            It is an enumerator defined by the LldpChassisIdSubtype object from IEEE 802.1AB 
+            MIB.
+          type: string
+          x-enum:
+            port_component:
+              x-field-uid: 1
+            network_address:
+              x-field-uid: 2
+            chassis_component:
+              x-field-uid: 3
+            mac_address:
+              x-field-uid: 4
+            interface_name:
+              x-field-uid: 5
+            local:
+              x-field-uid: 6
+            interface_alias:
+              x-field-uid: 7
+          x-field-uid: 5
+        neighbor_id:
+          description: >-
+            System generated identifier for the neighbor on the LLDP instance.
+          type: string
+          x-field-uid: 6
+        age:
+          description: >-
+            Age since discovery in seconds.
+          type: integer
+          format: int64
+          x-field-uid: 7
+        last_update:
+          description: >-
+            Seconds since last update received.
+          type: integer
+          format: int64
+          x-field-uid: 8
+        ttl:
+          description: >-
+            The time-to-live (TTL) in seconds is a mandatory TLV which indicates how long information from the neighbor 
+            should be considered valid.
+          type: integer
+          x-field-uid: 9
+        port_id:
+          description: >-
+            The Port ID is a mandatory TLV which identifies the port component of the endpoint identifier associated with 
+            the transmitting LLDP agent. If the specified port is an IEEE 802.3 Repeater port, then this TLV is optional.
+          type: string
+          x-field-uid: 10
+        port_id_type:
+          description: >-
+            This field identifies the format and source of the port identifier string. It is an enumerator defined by the
+            PtopoPortIdType object from RFC2922.
+          type: string
+          x-enum:
+            port_component:
+              x-field-uid: 1
+            network_address:
+              x-field-uid: 2
+            agent_circuit_id:
+              x-field-uid: 3
+            mac_address:
+              x-field-uid: 4
+            interface_name:
+              x-field-uid: 5
+            local:
+              x-field-uid: 6
+            interface_alias:
+              x-field-uid: 7
+          x-field-uid: 11
+        port_description:
+          description: >-
+            The binary string containing the actual port identifier for the port which this LLDP PDU was transmitted. The source 
+            and format of this field is defined by PtopoPortId from RFC2922.
+          type: string
+          x-field-uid: 12
+        management_address:
+          description: >-
+            The Management Address is a mandatory TLV which identifies a network address associated with the local LLDP agent, which 
```

Comment:
Is this really mandatory .  I had assumed only Chassis ID / Port ID and TTL are mandatory